README.md

```diff
@@ -1,14 +1,78 @@
-# Auth
+# Waterpark Server
```

Comment:
TODO: update this to relate to `auth`. Not sure if that's something we want to add to this PR.